sdk/src/index.ts

```diff
@@ -1,5 +1,40 @@
-export type { Project, OpenOptions, EmbedOptions } from './interfaces';
-export type { VM } from './VM';
+import {
+  connect,
+  embedGithubProject,
+  embedProject,
+  embedProjectId,
+  openGithubProject,
+  openProject,
+  openProjectId,
+} from './lib';
 
-import * as StackBlitzSDK from './lib';
-export default StackBlitzSDK;
+// Explicitly export public types (vs using `export * from './interfaces'`),
+// so that additions to interfaces don't become a part of our public API by mistake.
+export type {
+  Project,
+  ProjectOptions,
+  ProjectDependencies,
+  ProjectFiles,
+  ProjectSettings,
+  ProjectTemplate,
+  EmbedOptions,
+  OpenOptions,
+  OpenFileOption,
+  UiThemeOption,
+  UiViewOption,
+} from './interfaces';
+export type { FsDiff, VM } from './vm';
+
+// Export a single object with methods, for compatibility with UMD and CommonJS.
+// Ideally we would also have named exports, but that can create incompatibilities
+// with some bundlers, and microbundle doesn't support it:
+// https://github.com/developit/microbundle/issues/712
+export default {
+  connect,
+  embedGithubProject,
+  embedProject,
+  embedProjectId,
+  openGithubProject,
+  openProject,
+  openProjectId,
```

Comment:
This is technically the same as what we had before with

```
import * as StackBlitzSDK from './lib';
export default StackBlitzSDK;
```

EXCEPT that if we add a new export to `src/lib.ts`, it won't automatically end up as part of our public API.
This forces us to be more explicit and import-then-export it from `src/index.ts`. Maybe a bit verbose, but more intentional and thus safer.